### ISSUE TITLE
On IE11 Toggle button is missing keyboard focus border

### ISSUE THREAD

**Issue by adriantran** [OfficeDev/office-ui-fabric-react#4267]
### Bug Report

- __Package version(s)__: (current)
- __Browser and OS versions__: IE11

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes, for karma

Requested priority: Normal

Products/sites affected: TFS

#### Describe the issue:
One IE11, without any style reset, button has overflow hidden. So for the Toggle component, focus border is not shown.

https://jsfiddle.net/g5cdr73c/

#### Actual behavior:
No keyboard focus border

#### Expected behavior:
Rectangular keyboard focus border

**Comment by adriantran** [comment]
already fixed
